### PR TITLE
Add example Alpine Node Dockerfile

### PR DESCRIPTION
**Pull request by fubar**
As discussed in https://github.com/remind101/ssm-env/issues/20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,3 +55,68 @@ ENTRYPOINT ["/usr/local/bin/ssm-env", "-with-decryption"]
 ```
 
 Now, any command executed with the Docker image will be funneled through ssm-env.
+
+### Usage with Alpine Node Docker Images
```
Comment (russellballestrini) [inline README.md]: Would you mind making this section a subsection and title it, `Alpine Docker Image`

File: README.md [hunk 2]
```diff
@@ -55,3 +55,68 @@ ENTRYPOINT ["/usr/local/bin/ssm-env", "-with-decryption"]
...
+
+To use `ssm-env` with [Alpine Node](https://github.com/mhart/alpine-node) or other
+[Alpine Linux](https://alpinelinux.org/) Docker images, root certificates need to be added and the installation
+command differs, as shown in the `Dockerfile` below. Note that this example `Dockerfile` for Alpine Node also
+includes [dumb-init](https://github.com/Yelp/dumb-init) which is recommended when running Node in containers.
+
+```dockerfile
+FROM node:11-alpine
```
Comment (russellballestrini) [inline README.md]: Mind switching this image to the vanilla alpine image?

Comment (russellballestrini) [reply README.md]: https://hub.docker.com/_/alpine

File: README.md [hunk 3]
```diff
@@ -55,3 +55,68 @@ ENTRYPOINT ["/usr/local/bin/ssm-env", "-with-decryption"]
 ```
 
 Now, any command executed with the Docker image will be funneled through ssm-env.
+
+### Usage with Alpine Node Docker Images
+
+To use `ssm-env` with [Alpine Node](https://github.com/mhart/alpine-node) or other
```
Comment (russellballestrini) [inline README.md]: mind switching this url to `[Alpine](https://hub.docker.com/_/alpine)`

File: README.md [hunk 4]
```diff
@@ -55,3 +55,68 @@ ENTRYPOINT ["/usr/local/bin/ssm-env", "-with-decryption"]
...
+[Alpine Linux](https://alpinelinux.org/) Docker images, root certificates need to be added and the installation
+command differs, as shown in the `Dockerfile` below. Note that this example `Dockerfile` for Alpine Node also
+includes [dumb-init](https://github.com/Yelp/dumb-init) which is recommended when running Node in containers.
+
+```dockerfile
+FROM node:11-alpine
+
+# ...copy code, npm install etc.
```
Comment (russellballestrini) [inline README.md]: `# ...copy code`

File: README.md [hunk 5]
```diff
@@ -55,3 +55,68 @@ ENTRYPOINT ["/usr/local/bin/ssm-env", "-with-decryption"]
...
+
+# dumb-init: See https://github.com/Yelp/dumb-init
+RUN wget -O /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64
+RUN chmod +x /usr/local/bin/dumb-init
```
Comment (russellballestrini) [inline README.md]: This seems to confuse the tutorial, could we leave it out?

File: README.md [hunk 6]
```diff
@@ -55,3 +55,68 @@ ENTRYPOINT ["/usr/local/bin/ssm-env", "-with-decryption"]
...
+
+# ssm-env: See https://github.com/remind101/ssm-env
+RUN wget -O /usr/local/bin/ssm-env https://github.com/remind101/ssm-env/releases/download/v0.0.2/ssm-env
+RUN chmod +x /usr/local/bin/ssm-env
+
+# Alpine Node doesn't include root certificates which ssm-env needs to talk to AWS.
```
Comment (russellballestrini) [inline README.md]: I think this is true of most base Alpine images, please remove `Node`

**Comment by fubar** [comment]
@russellballestrini Updated as per your feedback

**Comment by russellballestrini** [comment]
@fubar thanks a bunch!

**Comment by fubar** [comment]
Thank you sir!

**Pull request merged**